articles/quickstarts/index.md

```diff
@@ -2,7 +2,7 @@
 title: Install the Microsoft Quantum Development Kit (QDK)
 description: How to install the Microsoft Quantum Development Kit for different environments.
```

Comment:
'How to get started'
In general, unless we are talking about installing a specific component of the QDK (extension, conda package..), we should use 'get the QDK' or 'get started with the QDK' instead of 'install'